neqo-client/src/main.rs

```diff
@@ -806,12 +814,19 @@ mod old {
         token: Option<Vec<u8>>,
         ciphers: Option<&[Cipher]>,
     ) -> Res<Option<Vec<u8>>> {
+        let quic_protocol = if args.alpn == "hq-27" {
+            QuicVersion::Draft27
+        } else {
+            QuicVersion::Draft28
+        };
+
         let mut client = Connection::new_client(
             origin,
-            &["hq-27"],
+            &["hq-28"],
```

Comment:
That's not the way the http/0.9 code in neqo-client currently works. It's triggered by the `-o` (--use-old-http) parameter, so default args.alpn is not right so old_client doesn't use it. I've opened #698, this could use some attention.